src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp

```diff
@@ -169,6 +169,48 @@ static void regclass_graph_PreProcessSteps(py::module m) {
     steps.def("reverse_channels", [](ov::preprocess::PreProcessSteps& self) {
         return &self.reverse_channels();
     });
+
+    steps.def(
+        "pad",
+        [](ov::preprocess::PreProcessSteps& self,
+           const std::vector<int>& pads_begin,
+           const std::vector<int>& pads_end,
+           float value,
+           ov::preprocess::PaddingMode mode) {
+            return &self.pad(pads_begin, pads_end, value, mode);
+        },
+        py::arg("pads_begin"),
+        py::arg("pads_end"),
+        py::arg("value"),
+        py::arg("mode"),
+        R"(
+            Adds padding preprocessing operation.
+            :param operation: Destination type. If not specified, type will be taken from model input's element type
```

Comment:
I suppose that we should use below format:
:param pads_begin: ...
:type pads_begins: ...
:param pads_end: ...
:type pads_end: ...
:param value: ...
:type value: ...
:param model: ...
:type model: ...
:return:  ...
:rtype: ...